python/ray/autoscaler/_private/util.py

```diff
@@ -724,6 +724,18 @@ def filter_placement_group_from_bundle(bundle: ResourceBundle):
     return demand_lines
 
 
+def get_constraints_report(lm_summary: LoadMetricsSummary):
```

Comment:
How about passing `lm_summary.request_demand` to the function instead? This allows us to mock data easier for unit testing.

Reply:
🆗